k8s/templates/studio-deployment.yaml

```diff
@@ -47,8 +47,8 @@ spec:
         env: {{ include "studio.sharedEnvs" . | nindent 8 }}
         - name: SEND_USER_ACTIVATION_NOTIFICATION_EMAIL
           value: "true"
-        - name: DATA_DB_HOST
-          value: {{ template "studio.fullname" . }}-frontend-db-conn-pool-service
+#        - name: DATA_DB_HOST
```

Comment:
could you change the value to `{{ .Values.postgresql.externalCloudSQL.proxyHostName | default (include "postgresql.fullname" .) }}` instead of disabling this here? thank you!

Reply:
@lyw07 Done, thanks! :)